zone/zone.h

```diff
@@ -343,6 +356,8 @@ class Zone
 	int	totalBS;
 	ZoneSpellsBlocked *blocked_spells;
 
+	ZoneBanishPoint* zone_banish_point;
```

Comment:
Please initialize this in the default constructor (Zone::Zone), not just filling it in at Zone::Init. Alternatively, don't make this a pointer, just a class member that can be referenced, and simply fill it in to all 0's with memset. I'd prefer the latter, since it's a simple structure and won't take up but a few bytes of memory to store. If you do that, change

`ZoneBanishPoint* GetZoneBanishPoint() { return zone_banish_point; }`

to

`ZoneBanishPoint& GetZoneBanishPoint() { return zone_banish_point; }`

and change the definition in the class to:

`ZoneBanishPoint zone_banish_point;`

and don't allocate new/delete if you change that

Reasoning:
Dynamics may have 'nullptr' for the 'Zone' class until booted, but for code quality's sake, we have encountered issues previously where code was added to worldserver.cpp in the zone process and the entire process crashes due to a null zone pointer during dynamic shutdown.